Cargo.toml

```diff
@@ -18,7 +18,7 @@ exclude     = [
 [dependencies]
 libc = { version = "0.2.60", features = [ "extra_traits" ] }
 bitflags = "1.1"
-cfg-if = "0.1.2"
+cfg-if = "0.1.10"
```

Comment:
Did you actually verify that 0.1.10 is the minimum version that works?

Reply:
I think I mentioned before that the error is still there on 0.1.9. I didn't try every version in between 0.1.2 and 0.1.9; it seems unlikely any of those work work if the same error is in both 0.1.2 and 0.1.9, but I guess it's not impossible. Is there an impact to jumping to 0.1.10?

Reply:
Well, given that 0.1.10 comes immediately after 0.1.9, I'd say that confirms it.  Too bad that the cfg-if crate doesn't maintain a CHANGELOG.